lib/App/Pinto/Command/add.pm

```diff
@@ -232,6 +234,26 @@ to work on those distributions, in the event that you need to patch them
 in the future.  Be aware that most distributions do not actually declare
 their development prerequisites.
 
+=item --verify
+
+=item -Z
+
+!! THIS OPTION IS EXPERIMENTAL !!
+
+Verify upstream distribution files before operating on them.  Verifies
+checksums and signatures where appropriate using the same mechanism as the
+C<audit> command.  If the verification fails, the process is aborted.
+Warnings about unknown or untrusted PGP keys are not considered fatal
+depending on the state of the acitve keyring.
+
+=item --strict
+
+!! THIS OPTION IS EXPERIMENTAL !!
+
+Modifies the C<--verify> option to make all warnings fatal and insisting that
```

Comment:
The intent was that `--strict` modified the behavior of `--verify` and its currently ignored  if `--verify` is omitted.  Definitely needs to be clarified.  I'll work something up when I work on better long names for these options, maybe, --verify-signatures and --verify-signatures-strictly.